sympy/parsing/latex/lark/latex_parser.py

```diff
@@ -9,12 +9,18 @@
 
 
 class LarkLatexParser:
-    def __init__(self, logger=False, print_debug_output=False, transform=True):
+    def __init__(self, logger=False, print_debug_output=False, transform=True, extends=None, overrides=None):
         grammar_dir_path = os.path.join(os.path.dirname(__file__), "grammar/")
 
         with open(os.path.join(grammar_dir_path, "latex.lark"), encoding="utf-8") as f:
             latex_grammar = f.read()
 
+        if extends is not None:
+            latex_grammar += "\n" + extends
+
+        if overrides is not None:
+            latex_grammar += "\n" + overrides
```

Comment:
How about just giving the option for the file path?

Reply:
We thought of that, and I think the idea was to make it as simple as possible for users, so @Upabjojr suggested doing it like this instead.

Reply:
I changed it to the file path option instead, because it's simpler and gives the user more flexibility.